scrapely/tool.py

```diff
@@ -1,9 +1,6 @@
 from __future__ import print_function
 import sys, os, re, cmd, shlex, optparse, json, pprint
-try:
-    from cStringIO import StringIO
-except ImportError:
-    from io import StringIO
+from six.moves import StringIO
```

Comment:
This is not 100% the same because in Python 2 six.moves.StringIO is StringIO.StringIO, not cStringIO.StringIO. But it doesn't matter for tool.py :)